crates/arroyo-types/Cargo.toml

```diff
@@ -10,3 +10,4 @@ serde = { version = "1.0", features = ["derive"] }
 arrow = { workspace = true }
 arrow-array = { workspace = true }
 async-trait = "0.1.74"
+blake3 = "1.5.0"
```

Comment:
do we need to add another hash dependency or can we use a built-in hash?

Reply:
Changed it to use `std::hash` and `base64`